planner/core/physical_plans.go

```diff
@@ -1058,7 +1058,11 @@ func (p *basePhysicalAgg) getAggFuncCostFactor() (factor float64) {
 		}
 	}
 	if factor == 0 {
-		factor = 1.0
+		if isMPP {
```

Comment:
Add comments and TODO for this if branch.

Reply:
done